runtime/lua/vim/lsp/diagnostic.lua

```diff
@@ -354,11 +354,12 @@ local function disable(bufnr)
   clear(bufnr)
 end
 
+--- @nodoc
```

Comment:
Because the method is underscored it will be private and we won't generate docs.
```suggestion
```